ale/base/data_naif.py

```diff
@@ -295,14 +295,20 @@ def sun_position(self):
         : (sun_positions, sun_velocities)
           a tuple containing a list of sun positions, a list of sun velocities
         """
-        sun_state, _ = spice.spkezr("SUN",
-                                     self.center_ephemeris_time,
-                                     self.reference_frame,
-                                     'LT+S',
-                                     self.target_name)
-        positions = 1000 * np.asarray([sun_state[:3]])
-        velocities = 1000 * np.asarray([sun_state[3:6]])
-        times = np.asarray([self.center_ephemeris_time])
+        times = [self.center_ephemeris_time]
+        positions = []
+        velocities = []
```

Comment:
Is there an advantage to initializing these up here and then appending to them later in python?